.github/workflows/ci.yaml

```diff
@@ -69,9 +57,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: stCarolas/setup-maven@v4
-      with:
-        maven-version: 3.8.1
```

Comment:
Were these changes intentional?

Reply:
I did not do them. I made a change to pom.xml file only.